### PR TITLE
Add tarred dataset doc for SC and SR

### PR DESCRIPTION
**Pull request by fayejf**
Add tarred dataset section for speech classification and speaker recognition in doc.

### CODE REVIEW

File: docs/source/asr/speaker_recognition/configs.rst [hunk 1]
```diff
@@ -47,6 +47,9 @@ An example SpeakerNet train and validation configuration could look like:
       batch_size: 32
       shuffle: False    # No need to shuffle the validation data
 
+      
+If you want to use tarred dataset, have a look at `Datasets Configuration <../configs.html#dataset-configuration>`__.
```
Comment (titu1994) [inline docs/source/asr/speaker_recognition/configs.rst:51]: If you would like to use ...

File: docs/source/asr/speaker_recognition/datasets.rst [hunk 2]
```diff
@@ -46,4 +46,12 @@ Each line in the manifest file describes a training sample - audio_filepath cont
...
\ No newline at end of file
+    {"audio_filepath": "<absolute path to dataset>/audio_file.wav", "duration": 3.9, "label": "speaker_id"}
+
+
+Tarred Datasets
+---------------
+
+Similarly to ASR, you can tar your audio files and use ASR Dataset class ``TarredAudioToSpeechLabelDataset`` (corresponding to the ``AudioToSpeechLabelDataset``) for this case.
+
+If you want to use tarred dataset, have a look at `ASR Tarred Datasets <../datasets.html#tarred-datasets>`__.
```
Comment (titu1994) [inline docs/source/asr/speaker_recognition/datasets.rst:57]: Would like to use

File: docs/source/asr/speech_classification/datasets.rst [hunk 3]
```diff
@@ -121,3 +121,11 @@ Preparing Custom Speech Classification Data
...
 
 Instead of :code:`text` entry in manifest, you need :code:`label` to determine class of this sample
+
+
+Tarred Datasets
+---------------
+
+Similarly to ASR, you can tar your audio files and use ASR Dataset class ``TarredAudioToClassificationLabelDataset`` (corresponding to the ``AudioToClassificationLabelDataset``) for this case.
+
+If you want to use tarred dataset, have a look at `ASR Tarred Datasets <../datasets.html#tarred-datasets>`__.
```
Comment (titu1994) [inline docs/source/asr/speech_classification/datasets.rst:131]: Would like to

**Review by titu1994 — APPROVED** [review]
Overall looks great. Minor comments

**Comment by fayejf** [comment]
@titu1994 Thanks! Updated. Will merge after the checks.

**Pull request merged**